charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml

```
@@ -33,7 +33,7 @@ spec:
       containers:
         - name: create
           {{- with .Values.controller.admissionWebhooks.patch.image }}
-          image: "{{.repository}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
+          image: "{{.registry}}/{{.repository}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
```

This is not backward-compatible, and will fail if the repository value from the chart's default `values.yaml` file is overridden. Consider something like this, to avoid the breaking change:
```suggestion
          image: "{{- if .registry -}}{{ .registry }}/{{- end -}}{{.repository}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
```

---

thank you @amirschw for the feedback!

I incorporated your change and rippled across the other files